src/emissions-workspace/emissions-checks.service.ts

```diff
@@ -57,6 +59,63 @@ export class EmissionsChecksService {
     return errorList;
   }
 
+  async invalidFormulasCheck(
+    payload: EmissionsImportDTO,
+    monitoringLocationId: string,
+  ): Promise<void> {
+    const formulaIdentifiers: string[] = [];
```

Comment:
It may be better if formulaIdentifiers here was a Set because there could be repeating formulaIds and then you'd be querying for the same formulaids in the below for loop. Let me know what you think.